components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -237,22 +221,136 @@ function OrgSettingsForm(props: { org?: OrganizationInfo }) {
+                        <Stack />
+                    </span>
+                    <span className="truncate">{parseDockerImage(props.image).repository}</span>
+                    &nbsp;&middot;&nbsp;
+                    <span className="flex-none w-16 truncate text-gray-500 dark:text-gray-500">
+                        {parseDockerImage(props.image).tag}
+                    </span>
+                </div>
+                <Button
+                    disabled={props.disabled}
+                    htmlType="button"
+                    type="transparent"
+                    className="text-blue-500"
+                    onClick={props.onClick}
+                >
+                    Change
+                </Button>
+            </div>
+        </InputField>
+    );
+}
+
+interface OrgDefaultWorkspaceImageModalProps {
+    globalDefaultImage: string | undefined;
+    settings: OrganizationSettings | undefined;
+    onClose: () => void;
+}
+
+function OrgDefaultWorkspaceImageModal(props: OrgDefaultWorkspaceImageModalProps) {
+    const [errorMsg, setErrorMsg] = useState("");
+    const [defaultWorkspaceImage, setDefaultWorkspaceImage] = useState(props.settings?.defaultWorkspaceImage ?? "");
+    const updateTeamSettings = useUpdateOrgSettingsMutation();
+
+    const handleUpdateTeamSettings = useCallback(
+        async (newSettings: Partial<OrganizationSettings>) => {
+            try {
+                await updateTeamSettings.mutateAsync({
+                    ...props.settings,
+                    ...newSettings,
+                });
+                props.onClose();
+            } catch (error) {
+                console.error(error);
+                setErrorMsg(error.message);
+            }
+        },
+        [updateTeamSettings, props],
+    );
+
+    return (
+        <Modal
+            visible
+            closeable
+            onClose={props.onClose}
+            onSubmit={() => handleUpdateTeamSettings({ defaultWorkspaceImage })}
+        >
+            <ModalHeader>Workspace Default Image</ModalHeader>
+            <ModalBody>
+                <Alert type="warning" className="mb-2">
+                    Warning: You are setting a default image for all workspaces within the organization.
+                </Alert>
+                {errorMsg.length > 0 && (
+                    <Alert type="error" className="mb-2">
+                        {errorMsg}
+                    </Alert>
+                )}
+                <div className="mt-4">
+                    <TextInputField
+                        label="Default Image"
+                        hint="Use any official or custom workspaces images from Docker Hub or any private container registries this Gitpod instance can access."
+                        placeholder={props.globalDefaultImage}
+                        value={defaultWorkspaceImage}
+                        onChange={setDefaultWorkspaceImage}
+                    />
```

Comment:
**question**: This is working great for any image from Docker Hub, but when using the full URL is returning an error. Does. it make sense to allow users to also paste examples like the following, regardless of the private registries settings, etc? Not sure, what's best practice for referencing container images? What do you think?

```sh
// Option 1 ✅ 
> gitpod/workspace-full

// Option 3 :x:
> docker.io/gitpod/workspace-full

// Option 3 :x:
> https://hub.docker.com/r/gitpod/workspace-full
```

Reply:
@gtsiolis You can treat this verify like `docker run <image_ref>`. And it's similer to `.gitpod.yml` we don't allow value like `https://hub.docker.com/r/gitpod/workspace-full` here too. 

cc @loujaybee  since this is a product decision

|images| works?|
|:-:|:-:|
|`docker.io/gitpod/workspace-full`|Works for me|
|`https://hub.docker.com/r/gitpod/workspace-full`| Should not work, you can verify also with `docker run <image_ref>`|

Reply:
Ah, ok. Looks good as is then. Thanks for clarifying, @mustard-mh. 🌮